sklearn/feature_selection/from_model.py

```diff
@@ -227,3 +231,6 @@ def partial_fit(self, X, y=None, **fit_params):
             self.estimator_ = clone(self.estimator)
         self.estimator_.partial_fit(X, y, **fit_params)
         return self
+
+    def _more_tags(self):
+        return {'allow_nan': True}
```

Comment:
I feel like this should be delegated to the estimator, so as to avoid the hack in `check_estimators_pickle`. WDYT @jnothman?

Reply:
Yes, let's return allow_nan accoring to the base estimator.